brave/brave-core
# Add pinned sites alert

Resolves https://github.com/brave/brave-browser/issues/1910

UI implementation: https://github.com/brave/brave-ui/pull/253 and https://github.com/brave/brave-ui/pull/263
## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
1.

- import from profile that has pinned sites
- go to rewards settings
- make sure that you see alert box in tip box
- reload site
- make sure that alert is gone

2. 
- import from profile that don't have pinned sites
- go to rewards settings
- make sure that alert box is not displayed

## Reviewer Checklist:
